fetools-app/src/pages/UnitConverter.jsx

```diff
@@ -27,15 +37,18 @@ function UnitConverter() {
     let finalSize = newSizeInPixels;
 
     // Check if newSizeInPixels exceeds the maximum allowed size for the preview (1000)
-    if (newSizeInPixels > 1000 && !alertShown) {
+    if (newSizeInPixels > 1000) {
       finalSize = 1000;
-      // Notify the user that the preview is capped (you might want to throttle this or change the UX for a smoother experience)
-      alert(
```

Comment:
Instead of alerting, let's simply display this note in the preview area instead of a preview. That way the user experience is not interrupted for someone who just wants to convert units and does not care about the preview.